### PR TITLE
fixed log messages for some states

### PR DESCRIPTION
**Pull request by VLGorskij**

### CODE REVIEW

File: lib/Sensors/Gateway.js [hunk 1]
```diff
@@ -353,7 +353,7 @@ Gateway.prototype.Control = function (attr, value) {
             sid:      this.sid,
             short_id: 0,
             data: {
-                temp_cfg:  parseInt(value,10),
```
Comment (GermanBluefox) [inline lib/Sensors/Gateway.js:356]: Why not the "number" type?

Comment (VLGorskij) [reply lib/Sensors/Gateway.js:356]: Hello. Look here, it seems to me that this is enough.
Снимок

Comment (VLGorskij) [reply lib/Sensors/Gateway.js:356]: ?

Comment (Apollon77) [reply lib/Sensors/Gateway.js:356]: This is just a type definition. if you want tzo make sure also a number is written you need the parseInt

**Comment by VLGorskij** [comment]
Good. I fixed it

**Comment by VLGorskij** [comment]
?

**Comment by GermanBluefox** [comment]
Thank you!

**Pull request merged**